aidatlu/main/tlu.py

```diff
@@ -17,7 +17,7 @@
 from aidatlu.hardware.ioexpander_controller import IOControl
 from aidatlu.hardware.trigger_controller import TriggerLogic
 from aidatlu.main.config_parser import TLUConfigure
-from aidatlu.main.data_parser import DataParser
+import aidatlu.main.data_parser as DataParser
```

Comment:
While this works, why import the whole module? Can you not import `interpret_data()`? If you need `_create_table()`, just move it inside `interpret_data()`. Keeps the code cleaner than using a module like a class.

Reply:
Yes, I dont even need `_create_table()`.